### PR TITLE
Docs: fix markdown in related meta data section

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/organizing/areas.markdown [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
-- docs: /docs/organizing/
-- docs: /docs/organizing/labels/
-  title: Labels
-- docs: /docs/organizing/categories/
-  title: Categories
-- docs: /docs/configuration/templating/#areas
-  title: Using areas in template
+  - docs: /docs/organizing/areas/
+    title: Areas
+  - docs: /docs/organizing/
```
Comment (klaasnicolaas) [inline source/_docs/organizing/areas.markdown:7]: No title?

File: source/_docs/organizing/categories.markdown [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
-- docs: /docs/organizing/floors/
-  title: Floors
-- docs: /docs/organizing/labels/
-  title: Labels
+   - docs: /docs/organizing/areas/
+   title: Areas
+   - docs: /docs/organizing/floors/
+   title: Floors
+   - docs: /docs/organizing/labels/
+   title: Labels
```
Comment (klaasnicolaas) [inline source/_docs/organizing/categories.markdown]: Indents going wrong

File: source/_docs/organizing/floors.markdown [hunk 3]
```diff
@@ -2,15 +2,15 @@
...
-- docs: /docs/organizing/
-- docs: /docs/organizing/labels/
-  title: Labels
-- docs: /docs/configuration/templating/#floors
-  title: Using floors in templates
-- docs: /voice_control/aliases/
-  title: Using floor alias for voice assistants
+  - docs: /docs/organizing/areas/
+    title: Areas
+  - docs: /docs/organizing/
```
Comment (klaasnicolaas) [inline source/_docs/organizing/floors.markdown:7]: No title?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_docs/organizing/areas.markdown (hunk 1 above)
Comment (c0ffeeca7) [reply source/_docs/organizing/areas.markdown:7]: in that case, it takes the page title

File: source/_docs/organizing/floors.markdown (hunk 3 above)
Comment (c0ffeeca7) [reply source/_docs/organizing/floors.markdown:7]: It takes the title page if no title is defined

File: source/_docs/organizing/categories.markdown [hunk 4]
```diff
@@ -2,12 +2,12 @@
...
-- docs: /docs/organizing/floors/
-  title: Floors
-- docs: /docs/organizing/labels/
-  title: Labels
+   - docs: /docs/organizing/areas/
+     title: Areas
+   - docs: /docs/organizing/floors/
+     title: Floors
+   - docs: /docs/organizing/labels/
+     title: Labels
```
Comment (klaasnicolaas) [inline source/_docs/organizing/categories.markdown]:
```suggestion
  - docs: /docs/organizing/areas/
    title: Areas
  - docs: /docs/organizing/floors/
    title: Floors
  - docs: /docs/organizing/labels/
    title: Labels
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**